src/connect.js

```diff
@@ -16,11 +16,18 @@
  * const result1 = connected1();
  * result1 is { x: 1, y: 2 }
  *
- * @param {Object} baseParams
- * @return {Function}
+ * @param {Object} baseParams * @return {Function}
  */
 function connect(params) {
-  // write code here
+  return (extraParams) => {
```

Comment:
Good, but you can write simply 
```suggestion
  return (extraParams) => ({ ...params, ... extraParams });
```
Also, you can add default value to parameters than you can remove check